go.mod

```diff
@@ -3,30 +3,31 @@ module github.com/tendermint/tendermint
 go 1.13
```

Comment:
why not bump the version to 1.14 too?

Reply:
Really we should

Reply:
I don't think we should bump in a minor version, will ask the team what they think.

Reply:
> I don't think we should bump in a minor version

Why?